lib/tbot/tbot.go

```diff
@@ -384,6 +394,7 @@ func (b *Bot) Run(ctx context.Context) (err error) {
 			svc.log = b.log.With(
 				teleport.ComponentKey, teleport.Component(componentTBot, "svc", svc.String()),
 			)
+			svc.statusReporter = statusRegistry.AddService(svc.String())
```

Comment:
I'm curious: why do we add the `statusReporter` field separately everywhere instead of simply adding it to the initializer above?

Reply:
iirc - there's a bit of a cyclical dependency in that you need svc initialised to be able to call svc.String() - we initialize the logger seperately for the same reason.